src/Framework/Sdk/SdkResolverContext.cs

```diff
@@ -39,8 +39,11 @@ public abstract class SdkResolverContext
         ///     Version of MSBuild currently running.
         /// </summary>
         /// <remarks>
-        ///     File version based on commit height from our public git repository. This is informational
-        ///     and not equal to the assembly version.
+        ///    <format type="text/markdown"><![CDATA[
+        /// ## Remarks
+        ///     
+        /// File version is informational and not equal to the assembly version.
+        /// ]]></format>
```

Comment:
```suggestion
        ///     File version is informational and not equal to the assembly version.
```

Do we need to include info about how the file version is calculated?